packages/insight/src/components/coin-list/coin-list.ts

```diff
@@ -48,13 +57,28 @@ export class CoinListComponent implements OnInit {
     }
   }
 
+  public sortBy(order: string) {
+    if(this.mostRecentOrderSelected && order === 'Most Recent') {
+      return this.txs;
+    } else if (!this.mostRecentOrderSelected && order === 'Oldest') {
+      return this.txs;
+    }
+
+    this.txs =  (order === 'Most Recent') ? 
+    _.sortBy(this.txs, ["height"]).reverse() 
+    : _.sortBy(this.txs, ["height"]);
+
+    this.mostRecentOrderSelected = (order === 'Most Recent');
+  }
+
   processData(data) {
     const txs = [];
     data.forEach(tx => {
       const { mintHeight, mintTxid, value, spentHeight, spentTxid } = tx;
-      txs.push({ height: spentHeight, spentTxid, value });
-      txs.push({ height: mintHeight, mintTxid, value });
+      txs.push({ height: spentHeight, spentHeight, spentTxid, value });
```

Comment:
I think it is not necessary to add `spentHeight` and `mintHeight`, they are already in `height`